package.json

```diff
@@ -6,7 +6,7 @@
     "url": "https://github.com/mendix/widgets-resources.git"
   },
   "scripts": {
-    "postinstall": "lerna bootstrap --hoist --strict",
+    "postinstall": "lerna bootstrap --hoist --strict && patch-package",
```

Comment:
let's rather wait for the fix in the upstream than merge this

Reply:
They published the new version containing my fix

Reply:
Fixed